faq/serverless-functions.mdx

```diff
@@ -184,3 +219,27 @@ solutions like Scaleway Object Storage.
 
 Currently, a new function instance will always start after each deployment, even if there is no traffic and the minimum
 scale is set to 0. This behavior is not configurable at this time.
+
+## How can I store data in my Serverless ressource?
+
+Serverless ressources are by default [stateless](/serverless/functions/concepts/#stateless), local storage is ephemeral.
+
+In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless ressources can be connected to ressources from the Scaleway ecosystem.
```

Comment:
```suggestion
In different use cases like saving results of analysis, exporting data etc... it can be important to save data. Serverless resources can be connected to resources from the Scaleway ecosystem.
```